compiler/ras/Debug.cpp

```diff
@@ -596,11 +596,10 @@ TR_Debug::vtrace(const char * format, va_list args)
    if (_file != NULL)
       {
       char buffer[256];
-      if(TR::Options::_traceFileLength && TR::IO::ftell(_file) > (TR::Options::_traceFileLength * (1 << 20)))
+      if ((0 != TR::Options::_traceFileLength) && (TR::IO::ftell(_file) > (((long)TR::Options::_traceFileLength) << 20)))
          {
          TR::IO::fseek(_file, 0, SEEK_SET); // rewind the trace file
-         sprintf(buffer, "Rewind trace file ...\n\n\n");
-         TR::IO::vfprintf(_file, buffer, args);
+         TR::IO::fprintf(_file, "Rewind trace file ...\n\n\n");
```

Comment:
Won't this result in `buffer` not getting the "Rewind trace file ...\n\n\n" string? `buffer` is used below.

Reply:
`buffer` was only receiving the format string, this just uses the literal directly.

Reply:
Ah I see, the contents of `buffer` isn't used by `getDiagnosticFormat` below.